### PR TITLE
Fix XCode scemes

### PR DESCRIPTION
**Pull request by MatkovIvan**
* [ ] Has `CHANGELOG.md` been updated?
* [x] Are tests passing locally?
* [x] Are the files formatted correctly?
* [ ] Did you add unit tests?
* [ ] Did you check UI tests on the sample app? They are not executed on CI.
* [ ] Did you test your change with either the sample apps that are included in the repository or with a blank app that uses your change?

## Description

Fix XCode scemes

### CODE REVIEW

File: SasquatchMac/SasquatchMac.xcodeproj/xcshareddata/xcschemes/SasquatchMacSwift-Preview-ContainingApp.xcscheme [hunk 1]
```diff
@@ -0,0 +1,91 @@
...
+            BlueprintName = "SasquatchMacSwift-Preview-ContainingApp"
+            ReferencedContainer = "container:SasquatchMac.xcodeproj">
+         </BuildableReference>
+      </BuildableProductRunnable>
+   </ProfileAction>
+   <AnalyzeAction
+      buildConfiguration = "Debug">
+   </AnalyzeAction>
+   <ArchiveAction
+      buildConfiguration = "Release"
```
Comment (annakocheshkova) [inline SasquatchMac/SasquatchMac.xcodeproj/xcshareddata/xcschemes/SasquatchMacSwift-Preview-ContainingApp.xcscheme]: I just wanted to try it

File: Sasquatch/Sasquatch.xcodeproj/xcshareddata/xcschemes/SasquatchSwift Extension.xcscheme [hunk 2]
```diff
@@ -105,20 +105,6 @@
                ReferencedContainer = "container:../AppCenterPush/AppCenterPush.xcodeproj">
             </BuildableReference>
          </BuildActionEntry>
-         <BuildActionEntry
-            buildForTesting = "YES"
-            buildForRunning = "YES"
-            buildForProfiling = "YES"
-            buildForArchiving = "YES"
-            buildForAnalyzing = "YES">
-            <BuildableReference
```
Comment (annakocheshkova) [inline Sasquatch/Sasquatch.xcodeproj/xcshareddata/xcschemes/SasquatchSwift Extension.xcscheme]: I just wanted to try it

File: Sasquatch/Sasquatch.xcodeproj/xcshareddata/xcschemes/SasquatchSwift Extension.xcscheme [hunk 3]
```diff
@@ -105,20 +105,6 @@
...
-               BuildableIdentifier = "primary"
-               BlueprintIdentifier = "B21A71DA1DD2A38C0044BB1C"
-               BuildableName = "CrashLibIOS.framework"
-               BlueprintName = "CrashLibIOS"
-               ReferencedContainer = "container:../CrashLib/CrashLib.xcodeproj">
-            </BuildableReference>
-         </BuildActionEntry>
```
Comment (Jamminroot) [inline Sasquatch/Sasquatch.xcodeproj/xcshareddata/xcschemes/SasquatchSwift Extension.xcscheme]: Testing?

Comment (Jamminroot) [reply Sasquatch/Sasquatch.xcodeproj/xcshareddata/xcschemes/SasquatchSwift Extension.xcscheme]: GH multililne comments are quite cool, but not quite handy yet :(
PS feedback resolved :D

**Pull request merged**